include/volume/rotating.hpp

```diff
@@ -10,7 +10,8 @@
 #define ROTATING_H
 
 #include <Eigen/Eigen>
-
+#include<boost/random.hpp>
```

Comment:
use `""` instead of `<>` as non standard header. Also to make sure proper formatting, ensure space between `#include` and `<`